### ISSUE TITLE
Storypa11y running against mdx files?

### ISSUE THREAD

**Issue by KBroughCode** [tobilen/storybook-addon-storypa11y#6]
Hi,

As the title suggests, wondering if StoryPa11y can run against mdx files? I have storybook running with all the components rendering as expected however when I try to run the accessibility tests I get a failure advising "storypa11y found 0 stories". 

I recently used Storypa11y in a POC and my configs and file structure are the exact same - the only difference is we're writing stories in mdx files instead of js. Any info or workarounds would be greatly appreciated as really keen on using this tool. 

Thanks,

**Comment by tobilen** [comment]
hey,

sorry for not getting back to you, must've overlooked the issue. Anyway, with the latest release mdx support should be built-in, since we're using the default storybook config.

Cheers
